lib/index.js

```diff
@@ -370,14 +426,28 @@ exports.getStorageClient = (ctx) => {
     const url = `${ctx.configuration.fusebit_storage_audience}/v1/account/${ctx.configuration.fusebit_storage_account_id}/subscription/${ctx.configuration.fusebit_storage_subscription_id}/storage/${ctx.configuration.fusebit_storage_id}`;
 
     return {
+        etag: null,
         get: async () => {
             const response = await Superagent.get(url)
                 .set('Authorization', `Bearer ${accessToken}`)
                 .ok((res) => res.status < 300 || res.status === 404);
+            this.etag = response.headers.etag;
```

Comment:
In arrow functions, the `this` is not bound, I don't think this line will do what you expect it to. Instead, you need to do something like this: 

```
let result = {
  get: async () => {
     // ...
     result.etag = response.body.etag; // Notice `result.` not `this.`. Also notice `response.body.etag`, not `response.headers.etag`
  }
};
return result;
```

Reply:
Also - storage APIs return the value of the etag in the body, `response.body.etag`

Reply:
Didn't realize `=>` does this, wow! I switched to good old functions and tested it out, it works. 

Regarding the etag, it's also in the header `etag: 'W/"bf21a9e8fbc5a3846fb05b4fa0859e0917b2202f"'` which is a bit more standard I think but sure I can use what's in the body.